src/platform/FreeBSD.cc

```diff
@@ -31,14 +31,16 @@
 #include "platform/FreeBSD.hh"
 #include "CommandLine.hh"
 #include "UserError.hh"
-
+#include<fcntl.h>
```

Comment:
fcntl.h should be included below, with the C header includes (which is where it was previously included, I think?)